filters/openpolicyagent/openpolicyagent.go

```diff
@@ -801,3 +819,13 @@ func (l *QuietLogger) Error(fmt string, a ...interface{}) {
 func (l *QuietLogger) Warn(fmt string, a ...interface{}) {
 	l.target.Warn(fmt, a)
 }
+
+func handleStatusErrors(status bundle.Status, failed chan error, prefix string) {
+	if len(status.Errors) > 0 {
+		failed <- fmt.Errorf("%s failed: %w", prefix, errors.Join(status.Errors...))
+	}
+
+	if status.Code != "" {
+		failed <- fmt.Errorf("%s failed: %s", prefix, status.Code)
```

Comment:
I suspect that in this case, `len(status.Errors) > 0`, i.e. you'll write two errors to the channel which means the second will never be shown. Can you verify that this is actually visible in tests?

Reply:
It seems more reliable to depend on status.Code for errors as for [these conditions](https://github.com/open-policy-agent/opa/blob/v0.62.0/plugins/bundle/status.go#L76-L96). Modified accordingly with if else which will avoid writing twice. Will do a test case as well.

Reply:
You will hide the actual error message which will lead to hard to debug situations. I‘d rather combine checking for errrors and the http status code (if set)